pkg/tsdb/elasticsearch/data_query.go

```diff
@@ -384,11 +386,16 @@ func processLogsQuery(q *Query, b *es.SearchRequestBuilder, from, to int64, defa
 	_ = addDateHistogramAgg(aggBuilder, bucketAgg, from, to, defaultTimeField)
 }
 
-func processDocumentQuery(q *Query, b *es.SearchRequestBuilder, from, to int64, defaultTimeField string) {
+func processDocumentQuery(q *Query, b *es.SearchRequestBuilder, from, to int64, defaultTimeField string, shouldAddStandardizedTimeField bool) {
 	metric := q.Metrics[0]
 	b.Sort(es.SortOrderDesc, defaultTimeField, "boolean")
 	b.Sort(es.SortOrderDesc, "_doc", "")
 	b.AddDocValueField(defaultTimeField)
+	if shouldAddStandardizedTimeField {
```

Comment:
Since I don't see other calls to `processDocumentQuery`, what do you think on leaving line 100 as it was, and add here:

```suggestion
	if isRawDataQuery(q) {
```

Or

```suggestion
    shouldAddStandardizedTimeField := isRawDataQuery(q)
	if shouldAddStandardizedTimeField {
```

Reply:
Good catch! I also like that now `processDocumentQuery` is handling all logic and `processQuery` doesn't need to care about `standardizedTimeField` 🤩